packages/core/src/wallet/metamaskBridge.ts

```diff
@@ -0,0 +1,34 @@
+import type { MetaMaskProvider } from '@consensys/get-starknet/dist/type';
+import detectEthereumProvider from '@metamask/detect-provider';
+
+async function hasSnapSupport(provider: any) {
```

Comment:
IMO, may be use MetaMaskProvider instead of any

Reply:
Done.

Reply:
resolved